package.json

```diff
@@ -8,14 +8,14 @@
 		"prebuild": "npm run prebuild:pancake && node src/helper/helper.js components",
 		"build:site": "./node_modules/.bin/cuttlebelle",
 		"build:autoprefix": "postcss src/assets/css/style.css -u autoprefixer -r && postcss src/assets/css/example.css -u autoprefixer -r",
-		"build:sass": "mkdir -p src/assets/css/ && node-sass --output-style compressed src/sass/style.scss src/assets/css/style.css && node-sass --output-style compressed src/sass/example.scss src/assets/css/example.css",
-		"build:sass:dev": "mkdir -p src/assets/css/ && node-sass --output-style compact --source-map true src/sass/style.scss src/assets/css/style.css && node-sass --output-style compact --source-map true src/sass/example.scss src/assets/css/example.css",
+		"build:sass": "npm run mkdirp src/assets/css/ && node-sass --output-style compressed src/sass/style.scss src/assets/css/style.css && node-sass --output-style compressed src/sass/example.scss src/assets/css/example.css",
```

Comment:
i think it's just mkdirp and not npm run mkdirp, since there is no mkdirp script right

Reply:
@sukhrajghuman Good pickup, i've updated it to run relative to the project